### PR TITLE
Add some information about Hubs

### PR DESCRIPTION
**Pull request by pmn**
Add some simple docs for Hubs

### CODE REVIEW

File: src/convos/hubs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+When Hubs are enabled, any team member can join the Hub to check on customer conversations. Responders will get a mention in a thread when notifications are sent. **When Hubs are enabled, Abbot will no longer send Direct Messages to responders.**
+
+Also, conversations can be snoozed or closed directly from the Hub by adding the :eyes: or :white_check_mark: emojis.
+
+### In short
+You can get in-chat help for Hubs. Type `@abbot help hubs` or `.help hubs` in a room with Abbot for a short guide on Hubs.
+
+## Setting up a Hub
+If you started using Abbot recently, we made a default Hub for you when you added Abbot to your Slack. We usually name this channel `#hub-customers`, although your administrator may have renamed it. If you're not sure if you have a Hub, you can see the list of Hubs for your workspace at https://app.ab.bot/settings/rooms?tab=Hubs.
```
Comment (analogrelay) [inline src/convos/hubs.md:17]:
```suggestion
If you started using Abbot recently, we made a default Hub for you when you added Abbot to your Slack. We usually name this channel `#hub-customers`, although your administrator may have renamed it. If you're not sure if you have a Hub, you can see the list of Hubs for your workspace [in your Rooms list](https://app.ab.bot/settings/rooms?tab=Hubs).
```

No ugly URL in the prose pls :)

**Review by analogrelay — APPROVED** [review]

File: src/convos/hubs.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+uid: hubs
+---
+
+# Hubs
+
+A Hub is a special channel in your Slack that Abbot controls to help manage customer conversations. Abbot shares new customer conversations in the Hub, plus an ongoing summary of these conversations. Your team can discuss and plan in threads connected to these messages, away from shared customer channels. Hubs reduce the need to visit Abbot's website, because Abbot shares all the important information in the Hub.
+
+When Hubs are enabled, any team member can join the Hub to check on customer conversations. Responders will get a mention in a thread when notifications are sent. **When Hubs are enabled, Abbot will no longer send Direct Messages to responders.**
```
Comment (dahlbyk) [inline src/convos/hubs.md:9]: > any team member can join

Technically a Hub could be private and require an invite

File: src/convos/hubs.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Also, conversations can be snoozed or closed directly from the Hub by adding the :eyes: or :white_check_mark: emojis.
+
+### In short
+You can get in-chat help for Hubs. Type `@abbot help hubs` or `.help hubs` in a room with Abbot for a short guide on Hubs.
+
+## Setting up a Hub
+If you started using Abbot recently, we made a default Hub for you when you added Abbot to your Slack. We usually name this channel `#hub-customers`, although your administrator may have renamed it. If you're not sure if you have a Hub, you can see the list of Hubs for your workspace [from the Rooms page](https://app.ab.bot/settings/rooms?tab=Hubs).
```
Comment (dahlbyk) [inline src/convos/hubs.md:17]: > we made a default Hub for you when you added Abbot to your Slack

Guess we should implement this 😁

**Review by dahlbyk — COMMENTED** [review]
A few notes/nits, but this is a great start

**Pull request merged**